crates/bevy_input/src/gamepad.rs

```diff
@@ -66,40 +322,118 @@ pub enum ButtonSettingsError {
     },
 }
 
-#[cfg(feature = "serialize")]
-use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
+/// Gamepad [`bundle`](Bundle) with the minimum components required to represent a gamepad.
+#[derive(Bundle, Debug)]
+pub struct MinimalGamepad {
```

Comment:
I think this should just be GamepadBundle instead of MinimalGamepad, but instead of doing that, I'm thinking we should just skip right to the new Required Components:

```rust
#[derive(Component)]
#[require(GamepadSettings)]
struct Gamepad {
}
````

Note that the example above excludes GamepadAxes and GamepadButtons to be in line with my proposed merger into the Gamepad component.